articles/libraries/lock/v10/index.md

```diff
@@ -61,14 +61,24 @@ If you are using browserify or webpack to build your project and bundle its depe
 
 ### 1. Initializing Lock
 
-First, you'll need to initialize a new `Auth0Lock` object, and provide it with your Auth0 client ID (the unique client ID for each Auth0 client app, which you can get from the [management dashboard](${manage_url})) and your Auth0 domain (i.e. `jeffstest.auth0.com`).
+First, you'll need to initialize a new `Auth0Lock` or `Auth0LockPasswordless` object, and provide it with your Auth0 client ID (the unique client ID for each Auth0 client app, which you can get from the [management dashboard](${manage_url})) and your Auth0 domain (i.e. `jeffstest.auth0.com`). Both `Auth0Lock` and `Auth0LockPasswordless` have the same public methods.
```

Comment:
First, you'll need to initialize a new `Auth0Lock` object (or an `Auth0LockPasswordless` object, if you are going to be implementing passwordless authentication).